ee/src/main/java/org/jboss/as/ee/subsystem/ManagedThreadFactoryAdd.java

```diff
@@ -52,13 +50,16 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
         final int priority = ManagedThreadFactoryResourceDefinition.PRIORITY_AD.resolveModelAttribute(context, model).asInt();
 
         final ManagedThreadFactoryService service = new ManagedThreadFactoryService(name, jndiName, priority);
-        final ServiceBuilder<ManagedThreadFactoryImpl> serviceBuilder = context.getServiceTarget().addService(ConcurrentServiceNames.getManagedThreadFactoryServiceName(name), service);
+        final ServiceBuilder serviceBuilder = context.getCapabilityServiceTarget()
```

Comment:
Unless you want to refactor ManagedThreadFactoryService to support installation as a MSC 1.4.x service, you will need to use:
``` java
ServiceBuilder<ManagedThreadFactory> serviceBuilder = context.getCapabilityServiceTarget().addCapability(ManagedThreadFactoryResourceDefinition.CAPABILITY, service);
```

Reply:
done